modules/local/make_blast_fasta/main.nf

```diff
@@ -0,0 +1,42 @@
+process MAKE_BLAST_FASTA {
+    // conda "${moduleDir}/environment.yml"
+    // container "wave.seqera.io/wt/f0df4f3f12cd/wave/build:make_blast_fasta--b4fc6a3e025d3533"
+
+    input:
+    tuple val(meta), path(txt)
+
+    output:
+    tuple val(meta) , path('*.fasta')   , emit: blast_contigs_fasta
+    path "versions.yml"             , emit: versions
+
+    script:
+    def kfinal_contigs = task.ext.kfinal_contigs 
+    def viral_contig_list = task.ext.viral_contig_list
+    def blast_contigs_fasta = task.ext.blast_contigs_fasta
```

Comment:
The benefit of local variables is when you combine them with tertiary or elvis operators, if you simply wish to use the `task.ext.kfinal_contigs` then it's better to embed them in the `script` section directly.